docs/findings/el9.md

```diff
@@ -0,0 +1,332 @@
+
+```{eval-rst}
+.. note::
+   ``watchmaker``'s automation-content does not have the capability of ensuring that:
+
+   * The Pillar-parameter's ``ash-linux:lookup:root-mail-dest`` value is set to a valid email destination
+   * Even if the ``ash-linux:lookup:root-mail-dest`` value `is` set to a valid email destination, forwarding to that destination will actually `function`
+
+```
+
+# Ensure Chrony is only configured with the server directive
+
+**Condtionally-valid Finding:**
+
+Setup of the `chrony` time-synchronization system can be very site-specific. In fact, some sites may choose not to set it up, at all, due to having other methods for ensuring that their hosts' time is kept properly-synchronized with an authoritative source. By default, `watchmaker` will make no changes to the configuration of the `chrony` time-synchronization service unless one sets the `ash-linux:lookup:use-ntp` Pillar parameter to `True`. If set to `True`, `watchmaker` will attempt to close this finding:
+
+* If one further defines the `ash-linux:lookup:ntp-servers` Pillar-parameter to a list of NTP servers, `watchmaker` will close the finding by configuring the `chrony` service to use that list of servers
+* If one fails to define the `ash-linux:lookup:ntp-servers` Pillar-parameter `watchmaker` will close the finding by configuring the `chrony` service to a default list of servers (the per-vendor "pool" NTP servers maintained by the [Network Time Protocol (NTP) Project](https://ntp.org))
+
+# Enable SSH Server firewalld Firewall Exception
+
+**Invalid Finding:**
+
+This finding may be triggered if only the `ssh` _ports_ are scanned for. The `watchmaker` hardening routines ensure that a broad-scoped (i.e., "allow from all") firewalld exception is made for the `ssh` _service_. The implementation-difference may be seen by comparing the outputs of `firewall-cmd --list-services`
+
+```
+# firewall-cmd --list-services | sed 's/\s\s*/\n/g' | grep ssh
+ssh
+```
+
+and `firewall-cmd --list-ports`:
+
+```
+# firewall-cmd --list-ports | grep ^22
+22/tcp
+```
+
+Watchmaker's implementation will show up only in the output of the former. Some scanners may only expect the execption to show up in the latter.
+
+# Enable Certmap in SSSD
+
+**Expected Finding:**
+
+Because configuration of the `sssd` service to perform SmartCard-based authentication is an inherently-local configuration-task (and because no, suitable testing environment has been provided to this project-team to prototype against), `watchmaker` makes no attempt to configure `sssd` service to perform SmartCard-based authentication.
```

Comment:
```suggestion
Because configuration of the `sssd` service to perform SmartCard-based authentication is an inherently-local configuration-task (and because no suitable testing environment has been provided to this project-team to prototype against), `watchmaker` makes no attempt to configure `sssd` service to perform SmartCard-based authentication.
```